source/shared/cpp/ObjectModel/ParseContext.cpp

```diff
@@ -241,4 +242,48 @@ namespace AdaptiveSharedNamespace
         AdaptiveSharedNamespace::InternalId invalidId;
         return std::move(invalidId);
     }
+
+    ContainerStyle ParseContext::GetParentalContainerStyle() const
+    {
+        return (m_parentalContainerStyles.size())? m_parentalContainerStyles.back() : ContainerStyle::None;
```

Comment:
>( [](start = 15, length = 1)

[nit] Remove extra set of parenthesis

Reply:
thanks

---
In reply to: [258671656](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r258671656) [](ancestors = 258671656)